charts/external-es-proxy/templates/external-es-proxy-configmap.yaml

```diff
@@ -2,6 +2,7 @@
 ## Elasticsearch Proxy ConfigMap ##
 ###################################
 {{- if or (eq .Values.global.plane.mode "data") (eq .Values.global.plane.mode "unified") }}
+{{- $authCacheEnabled := .Values.authCache.enabled }}
```

Comment:
Nit: Again, I think the template is easier to read if we just check the helm value directly and skip creating a variable for it. This is not a critically important detail though.

Reply:
it was referenced in 3 different places in the same template thats why we thought of doing it